### ISSUE TITLE
When Using Gettext Docs Should Mention I18n::Gettext::Helpers Requirement

### ISSUE THREAD

**Issue by sshaw** [svenfuchs/i18n#318]
Doing so prevents source strings with a period from being translated. This is with 0.7.0:

```
~/ruby/i18n-test >cat /tmp/bs.rb
require "i18n"

I18n::Backend::Simple.include(I18n::Backend::Gettext)
# Fake load_po call
I18n.backend.store_translations(:pt, "See you soon." => "Até breve.")
I18n.locale = :pt
puts I18n.t "See you soon."
puts I18n.t "See you soon.", :separator => "___"

~/ruby/i18n-test >ruby /tmp/bs.rb
translation missing: pt.See you soon
Até breve.
```

Probably need a Gettext specific backend that does not require using `I18n::Backend::Simple`. There one can avoid the default call to `lookup` which calls `I18n.normalize_keys`. Or, you can allow for a `nil` or `false` separator value that will skip the `split`ting.

**Comment by sshaw** [comment]
Workaround is to use the methods in `I18n::Gettext::Helpers` instead of `I18n.t`.

**Comment by radar** [comment]
@sshaw would you like to add that documentation patch?

**Comment by sshaw** [comment]
> @sshaw would you like to add that documentation patch?

See #351

**Comment by radar** [comment]
Fixed by #351.
